atomate/vasp/firetasks/run_calc.py

```diff
@@ -241,6 +242,60 @@ def run_task(self, fw_spec):
         runner.run(path_dir=os.getcwd())
 
 
+@explicit_serialize
+class RunAmset(FiretaskBase):
+    """
+    Run Amset directly. Requires vasprun.xml to be in current dir.
+
+    Required params:
+        material_params (dict): the static dielectric constant (epsilon_s) is
```

Comment:
Tell people where to find more information about what goes inside this dictionary. Very unclear what else can go in and what happens if more or less info is specified